engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -3057,3 +3107,30 @@ enum EnabledState {
   /// The node is disabled.
   disabled,
 }
+
+extension SemanticsValidationResultExtension on ui.SemanticsValidationResult {
```

Comment:
Why does this use an extension instead of  directly in the enum class?

Reply:
The enum class is public. We don't want web-specific details to be published through `dart:ui` types. However, this is gone now. Originally, I had `toAriaValue` used in multiple places, but then collapsed it to one, so I don't need it to be a shared function anymore.